### PR TITLE
Update email/text to accept custom name and questions [closes #116, closes #136]

### LINKED ISSUE #116: Customise email content

### LINKED ISSUE #136: Add in customised names and questions in email - see description
Aware of time spent and within time constraint:
"Well done!
Looks like you are on track to achieve your ultimate dream goal :D
Keep it up as consistency is the key!"

Unaware of time spent and not within time constraint:
"Very good! You have shown good discipline today :D
It seems that you were not aware of how much time you lost on the distraction site.
How would performance change if you are more aware of how your time is spent?
What system can you implement  to help you achieve that?
"

Aware of time spent and not within time constraint:
"Not bad, you are aware of how much time you lost on the distraction site so  you knew that you still went over the allocated time.
Please think about:
What is already working in your system that you can build on?
What might 'help' look like?
"

Unaware of time spent and not within time constraint:
"Oops, not knowing how much time you spent on distraction sites... it seems like you are not on track to achieving you dream goal!
Think about:
Is this blacklist time constraint too stretchy for you right now? Consistent baby step is better than nothing!
In the beginning, how did you want it to be?
What is the best/worst thing that could happen if you continue this way?
What might 'help' look like?

"

### PR DESCRIPTION
**Pull request by bentljm**

### CODE REVIEW

File: client/src/components/Settings.js [hunk 1]
```diff
@@ -130,9 +130,9 @@ export default class Settings extends React.Component {
       type: 'POST',
       url: `api/users/${this.state.profile.user_id}/sendNotification`,
       contentType: 'application/json',
-      data: JSON.stringify({ address: this.state.reminderAddress }),
-      success: (data) => {
-        console.log('SUCCESS: SENT NOTIFICATIONS', data);
+      data: JSON.stringify({address: this.state.reminderAddress, name: this.state.profile.given_name}),
+      success: function (data) {
```
Comment (SherryH) [inline client/src/components/Settings.js]: It is better to use ES6 syntax to reduce linting errors. e.g. arrow function, single quotes

File: server/controllers/notificationControllers.js [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
-  	var userEmail = req.body.address;
-  	sendEmail(userEmail);
-  } else {
-  	var userText = req.body.address;
-  	sendText(userText);
-  }
-  console.log("SENT NOTIFICATIONS TO", userEmail || userText);
+	var auth0_id = req.params.auth0_id;
+	var name = req.body.name;
+	db.Reflection.findAll({where: {auth0_id: auth0_id}}).then(function (reflections) {
```
Comment (SherryH) [inline server/controllers/notificationControllers.js]: The reflections DB is empty in the beginning. It saves a pair of Q&A from the user input. 
Our generic Questions would be hardcoded somewhere in the code e.g. in a Question object.

File: client/src/components/Settings.js (hunk 1 above)
Comment (bentljm) [inline client/src/components/Settings.js]: changed line 134 to ES6 syntax

File: server/controllers/notificationControllers.js (hunk 2 above)
Comment (bentljm) [inline server/controllers/notificationControllers.js]: the generic question is hardcoded in emailNotifications.js "reflections: reflections || "How well do you think you spent your time today?"

Comment (bentljm) [inline server/controllers/notificationControllers.js]: I can add more questions here

File: client/src/components/Settings.js [hunk 3]
```diff
@@ -130,9 +130,9 @@ export default class Settings extends React.Component {
       type: 'POST',
       url: `api/users/${this.state.profile.user_id}/sendNotification`,
       contentType: 'application/json',
-      data: JSON.stringify({ address: this.state.reminderAddress }),
+      data: JSON.stringify({address: this.state.reminderAddress, name: this.state.profile.given_name}),
       success: (data) => {
-        console.log('SUCCESS: SENT NOTIFICATIONS', data);
+        console.log("SUCCESS: SENT NOTIFICATIONS");
```
Comment (SherryH) [inline client/src/components/Settings.js]: single quote please

Comment (bentljm) [inline client/src/components/Settings.js]: changed

**Pull request merged**